test/sqlsrv/srv_048_stream_nvarchar.phpt

```diff
@@ -0,0 +1,42 @@
+$sql = "INSERT INTO #Table VALUES (N'$data')";
+$stmt = sqlsrv_query($conn, $sql);
+if( !$stmt ) { die( print_r( sqlsrv_errors(), true)); }
+
+// Fetch
+$sql = "SELECT * FROM #Table";
+$stmt = sqlsrv_query($conn, $sql);
+if( !$stmt ) { die( print_r( sqlsrv_errors(), true)); }
+
+// Compare output
+$row = sqlsrv_fetch_array($stmt);
+echo ($row['c1'] == $data) ? "True\n" : "False\n";
```

Comment:
No trimming necessary? Nice. :) Just curious, have you tried using '===' for comparison?